lms/templates/registration/password_reset_confirm.html

```diff
@@ -32,6 +32,14 @@
                 'errorMessage': js_escaped_string(err_msg) if err_msg else '',
             },
         )}
+    % elif  user_exist:
+    <div class="status submission-success">
+        <h4 class="message-title">${_("User already login in this session")}</h4>
+        <ul class="message-copy">
+            You are already signed in on this browser. Please either log out from the current browser and login again to reset your password.</br/>
+            Or open the url in a new browser window to proceed for your password Reset.<br/>
```

Comment:
`You are already logged in. Please logout and try again or change browser.`

Lets keep it short